packages/varnish/src/Footer.tsx

```diff
@@ -0,0 +1,51 @@
+import { Link } from '@mui/material';
+import * as React from 'react';
+
+import { Color, color as varnishColor } from './colors';
+
+export type LayoutVariant = 'left-aligned' | 'center-aligned';
+
+export type FooterVariant = 'default' | 'dark';
+
+interface Props {
+    variant?: FooterVariant;
+    children?: React.ReactNode | React.ReactNode[];
+    layout?: LayoutVariant; // TODO: Add layout understanding to new footer (Investigate Grid?)
+    backgroundColor?: Color;
+    className?: string;
+}
+
+export class Footer extends React.PureComponent<Props> {
```

Comment:
does this need to be a class?
can we convert to react functional component
`export const Footer = (props: Props) => {...`